src/test/java/com/st/utopia/cancellationservice/CancellationServiceApplicationTests.java

```diff
@@ -0,0 +1,16 @@
+package com.st.utopia.cancellationservice;
```

Comment:
I suggest removing this class; your tests should go in test classes (and methods) with names reflecting the code being tested, so none will go here, and our CI static-analysis tools will object to the empty method here (both being an empty method and being a test method without any assertions).